docs/TuxActionStores.md

```diff
@@ -0,0 +1,107 @@
+#Tux ActionStores
+##Table of Contents
+<ol>
+  <li><a href="#Premise">Premise</a></li>
+  <li><a href="#Implementation">Implementation</a></li>
+    <ol style="list-style-type:upper-alpha">
+      <li><a href="#Requiring-ActionStores">Requiring ActionStores</a></li>
+      <li><a href="#Creating-Action-Stores">Creating ActionStores</a>
+      </li>
+      <li><a href="#TuxActionStores-Complete-Example">Tux ActionStores Complete Example</a></li>
+    </ol>
+</ol>
+## <a id="Premise"></a>Premise [#](#Premise)
+>Tux `ActionStores` is a module providing the interface for creating stores that are pre-packaged with a 'register' convenience method for automatically registering the store with actions.
+
+## <a id="Implementation"></a>Implementation [#](#Implementation)
+### <a id="Requiring-ActionStores"></a>Requiring Action Stores [#](#Requiring-Action-Stores)
+
+The interface for creating `ActionStores` is exposed via a constructor function:
+
+```
+  var createActionStore = require('tux/Stores/ActionStores');
```

Comment:
`require('tux/Stores/ActionStores')` actually provides an object with a method called `createStore`.  Thus this line should be:

```
    `var actionStore = require('tux/Stores/ActionStores')`
```